cg/cli/reset.py

```diff
@@ -14,25 +19,48 @@ def reset_cmd():
     pass
 
 
-@reset_cmd.command()
-@click.option("-c", "--case_id", help="internal case id, leave empty to process all")
+@reset_cmd.command("observations")
+@ARGUMENT_CASE_ID
 @click.pass_obj
-def observations(context: CGConfig, case_id: Optional[str]):
-    """Reset observation links from an analysis to LoqusDB."""
+def observations(context: CGConfig, case_id: str):
+    """Reset observation links from an analysis to Loqusdb for a specific case."""
+
     status_db: Store = context.status_db
-    observations_uploaded: Iterable[models.Family]
-    if case_id:
-        observations_uploaded = [status_db.family(case_id)]
-    else:
-        observations_uploaded = status_db.observations_uploaded()
+    case: models.Family = context.status_db.family(case_id)
 
-    for case_obj in observations_uploaded:
-        LOG.info("This would reset observation links for: %s", case_obj.internal_id)
+    if not case or case.data_analysis not in LOQUSDB_SUPPORTED_PIPELINES:
```

Comment:
I think this logic should be in a function in the cg/cli/upload/observations.py. It is almost already part of the function: get_observations_case. Do you think we can make it a more general function if we send in the Query as an argumnet or make a swith and make the log msg a bit more general? If we make it a separate function we can reuse the logic in both places and unit test it once.

Reply:
Yes, moved it to an utils file in observation and made it more general now :)